.github/workflows/tag-publish.yaml

```diff
@@ -116,6 +116,15 @@ jobs:
             "${{ steps.image-name.outputs.tagged }}" \
             "${{ steps.image-name.outputs.name }}:latest"
 
+      # if: ${{ github.event_name != 'pull_request' }}
+      # npm publish --dry-run --workspace packages
+      # yarn workspaces foreach --no-private npm publish --tolerate-republish
+      - name: Publish packages to NPM registry on merge commits
+        working-directory: packages
+        run: |
```

Comment:
We'll need to build the packages here before publishing them. Currently, only the package.json file is being published.